rosdep/python.yaml

```diff
@@ -689,6 +689,18 @@ python-evdev:
     vivid: [python-evdev]
     wily: [python-evdev]
     xenial: [python-evdev]
+python-expiringdict:
+  debian:
+    jessie: [python-expiringdict]
+    sid: [python-expiringdict]
+    stretch: [python-expiringdict]
+  fedora:
+    pip:
+      packages: [expiringdict]
+  ubuntu:
+    xenial: [python-expiringdict]
+    yakkety: [python-expiringdict]
```

Comment:
It's my bad I thought we needed to call out versions when it's not available for some of the currently supported platforms (e.g. trusty for this package). can you confirm @tfoote that it's fine to collapse the rules anyway?